Corona-Warn-App/src/main/java/de/rki/coronawarnapp/srs/core/server/SrsAuthorizationServer.kt

```diff
@@ -134,33 +136,35 @@ class SrsAuthorizationServer @Inject constructor(
         }
     }
 
-    private suspend fun errorArgs(errorCode: ErrorCode): Array<Any> = when (errorCode) {
-        ErrorCode.SUBMISSION_TOO_EARLY -> arrayOf(
-            appConfigProvider
-                .currentConfig
-                .first()
-                .selfReportSubmission
-                .common
-                .timeBetweenSubmissionsInDays
-                .toDays()
-        )
-
-        ErrorCode.TIME_SINCE_ONBOARDING_UNVERIFIED,
-        ErrorCode.MIN_TIME_SINCE_ONBOARDING -> {
```

Comment:
Are these going to be added back when you remove HTTP logging?

Reply:
this just a refactoring , it is a common function `ErrorCode.errorArgs()`

Reply:
see https://github.com/corona-warn-app/cwa-app-android/pull/5753/files#diff-0d3cedb31b14d57883b52c3cf3df13de9dad38979bcc8a32626aee0755f1ca48R152